<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>272</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border-radius: 5px;
	font: 16px &quot;Poppins&quot;;
	background-color: #1a759f;
	color: white;
	font-weight: bold;
}</string>
  </property>
  <widget class="QPushButton" name="btn10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>174</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>3</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>60</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn13">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>231</y>
     <width>83</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #266676;
color: rgb(255, 255, 255);
font-size: 28px;</string>
   </property>
   <property name="text">
    <string>⌫</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>3</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn9">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>117</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn14">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>231</y>
     <width>81</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #266676;
color: rgb(255, 255, 255);
font-size: 28px;</string>
   </property>
   <property name="text">
    <string>⇨</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>117</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn12">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>174</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>117</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>60</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn11">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>174</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
